src/openzaak/components/documenten/tests/admin/test_eiocanonical_inline_audittrail.py

```diff
@@ -136,3 +137,36 @@ def test_eio_add(self):
 
         new_data = audittrail.nieuw
         self.assertEqual(new_data["identificatie"], "12345")
+
+    def test_eio_add_no_version(self):
+
+        get_response = self.app.get(self.change_url)
+
+        response = get_response.form.submit()
+        self.assertEqual(response.status_code, 200)
+
+        version_form = response.context["inline_admin_formsets"][0].forms[0]
+
+        self.assertEqual(
+            version_form.errors["bronorganisatie"], [_("This field is required.")]
+        )
+        self.assertEqual(
+            version_form.errors["creatiedatum"], [_("This field is required.")]
+        )
+        self.assertEqual(version_form.errors["titel"], [_("This field is required.")])
+        self.assertEqual(version_form.errors["auteur"], [_("This field is required.")])
+        self.assertEqual(version_form.errors["taal"], [_("This field is required.")])
+        self.assertEqual(
+            version_form.errors["bestandsomvang"], [_("This field is required.")]
+        )
+        self.assertEqual(version_form.errors["inhoud"], [_("This field is required.")])
+        self.assertEqual(
```

Comment:
It might also be good to add an assertion to verify that no object was created as a sanity check